src/DetailsPage/DetailsPage.js

```diff
@@ -2,66 +2,135 @@ import React, {Component} from 'react'
+    // console.log('this.state.movieInfo.id', this.state.movieInfo.id)
+    
     const ratingOptions = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map(number => (
-    <option key={number} value={number}>{number}</option>
-    ))
-     const ratedMovie = this.findRating()
-
-     const findMovieIRated = this.props.userRatings.find(movie => movie.movie_id === this.props.movie.id)
-
+      <option key={number} value={number}>{number}</option>
+      ))
+      
+      const comments = this.state.movieComments.map(movie => {
+      return (<div>
+              {movie.author}
+              {movie.comment}
+            </div>)
+      })
+      //  const ratedMovie = this.findRating()
+      
+      const findMovieIRated = this.props.userRatings.find(movie => movie.movie_id === this.state.movieInfo.id)
+      console.log('findMovieIRated', findMovieIRated)
+      
     const displayUserRating = findMovieIRated ? <p><span>My Rating:</span> {findMovieIRated.rating.toFixed(1)}</p> : <p>You have not rated yet.</p>
     
 
 
     return (
+     
       <section className='DetailsPage'>
         <section className='backdrop-parent'>
           {this.props.error && <h1>{this.props.error}</h1>}
-          <img className='backdrop-img' src={this.props.movie.backdrop_path} alt={this.props.movie.title} />
-          {this.props.movie.tagline && <p className='tag-line'>{this.props.movie.tagline}</p>}
+          <img className='backdrop-img' src={this.state.movieInfo.backdrop_path} alt={this.state.movieInfo.title} />
+          {this.state.movieInfo.tagline && <p className='tag-line'>{this.state.movieInfo.tagline}</p>}
+
+          <form className='comment-form'>
+            <p>What did you think about {this.state.movieInfo.title}?</p>
+            <textarea name='userComment' value={this.state.userComment} placeholder='Enter comments here...' onChange={this.handleChange}></textarea>
+            <button type="button"  onClick={() => this.postUserComment(this.state.userComment)}>Submit Comment</button>
+          </form>
+          <section className='comments-section'>
+            {/* <p>{this.props.userName} says:</p> */}
+              <p>{comments}</p>
+          </section>
+
+
+
+
         </section>
         <article className='movie-details'>
-          <h1 className='movie-title'>{this.props.movie.title}</h1>
+          <h1 className='movie-title'>{this.state.movieInfo.title}</h1>
           <h2>Description:</h2>
-          {this.props.movie.overview && <p><span>Overview:</span> {this.props.movie.overview}</p>}
-          {this.props.movie.release_date && <p><span>Release Date:</span> {this.props.movie.release_date}</p>}
-          {this.props.movie.runtime > 0 && <p><span>Runtime:</span> {this.props.movie.runtime}</p>}
-          {this.props.movie.average_rating > 0 && <p><span>Average Rating:</span> {this.props.movie.average_rating.toFixed(1)}</p>}
+          {this.state.movieInfo.overview && <p><span>Overview:</span> {this.state.movieInfo.overview}</p>}
```

Comment:
This looks a bit difficult to read so I may try to play around with the formatting to see what looks best